### DISCUSSION TITLE
How to handle service endpoint address that contains a dot

### DISCUSSION THREAD

**Discussion by ahovland** [CoreWCF/CoreWCF#1349] [Q&A] [answered]
In my Program.cs my endpoint is set up like this:

```
app.UseServiceModel(serviceBuilder =>
{
	serviceBuilder.AddService<FsLicenseSvc>();
	serviceBuilder.AddServiceEndpoint<FsLicenseSvc, IFsLicenseSvc>(new BasicHttpBinding(BasicHttpSecurityMode.Transport), "/FsLicenseSvc.svc");
	var serviceMetadataBehavior = app.Services.GetRequiredService<ServiceMetadataBehavior>();
	serviceMetadataBehavior.HttpsGetEnabled = true;
});

```
My service is deployed to an Azure web app, but navigating to the service url in the browser results in HTTP 404.
If I change the service endpoint address to /FsLicenseSvc (i.e. remove the extension .svc) and redeploy, everything works fine.

My question is: What do I do to handle a service endpoint address that contains a dot (like /FsLicenseSvc.svc).

**Comment by mconnew** [comment]
There's nothing going on with that in CoreWCF, this is a supported scenario. I wonder if there's something special going on in with Azure web app service which is intercepting the request to hand to the .NET Framework handler base on the .svc extension

**Reply by ahovland** [reply, marked as answer]
The solution was to modify the staticContent section in web.config like this:

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
	<system.webServer>
		<staticContent>
			<remove fileExtension=".svc"/>
		</staticContent>
	</system.webServer>
</configuration>
```
